syncd/Syncd.cpp

```diff
@@ -63,6 +63,13 @@ Syncd::Syncd(
         SWSS_LOG_THROW("no context config defined at global context %u", m_commandLineOptions->m_globalContext);
     }
 
+    if (m_commandLineOptions->m_enableZmqSyncMode)
+    {
+        SWSS_LOG_NOTICE("zmq sync mode enabled via cmd line");
+
+        m_contextConfig->m_zmqEnable = true;
```

Comment:
>m_zmqEnable [](start = 25, length = 11)

Do you mean `m_zmqSyncMode` ? It is confusing if `sync` is not in the name.

Reply:
there is no m_zmqSyncMode in that class, m_zmqEnable was introduced previously in ContextConfig class without intention that it will be added as separate flag "syncd mode", since zmq is always synchronous, agree that those names could be confusing here, any proposal ?

Reply:
as your other comment should i drop "sync" from names ? and what about SAI_REDIS_SWITCH_ATTR_ZMQ_SYNC_MODE ?? actually "sync mode" should be referred as "redisdb sync mode" or "redis sync mode"